### ISSUE TITLE
Where does the output file get exported by default? (bug?) [P.S.: Probable Insomnia version upgrade issue -Vyoam]

### ISSUE THREAD

**Issue by jake-ruth-qm** [Vyoam/insomnia-plugin-postman-export#2]
Hello! 

Thanks for developing this plugin.

When looking through the code, it seems that multiple dialogues are supposed to appear when exporting, but I only get one. When I click on Export as a Postman Collection, I get a dialogue but then after clicking "Ok" I get no feedback and no location of the file output.

I tried with your script repo and that works, I'd love to get this functional with the plugin if possible :)

Thanks!

**Comment by Vyoam** [comment]
Thanks for the feedback @jake-ruth-qm 
**I see this too. Seems to be happening after a recent Insomnia release. Will try to take a look over the weekend.**
BTW the flow is that you will get 2 prompts for filters (mutually independent) then a last prompt for the destination folder path (I think I have hard-coded just the file name - NOT the folder - for now).

BTW, I hadn't thought about how these kind of issues can be smoothly tracked, as they are/will be related to Insomnia base rather than the plugin code, so any UTs in my own code will be useless. Will try to read up/think over it and have a plan for future too.

**Comment by jake-ruth-qm** [comment]
@Vyoam Awesome!

Thank you for the quick response. If you need any assistance let me know and I could look into it and throw up a PR this weekend if I have time :)

**Comment by jake-ruth** [comment]
Hey I looked into this, it's a bug on their end with the api:

https://github.com/Kong/insomnia/issues/3959

So I think once they fix it we'll be good!

**Comment by Vyoam** [comment]
Thanks @jake-ruth. I can also confirm that we have the same issue here - around the context.app.prompt() calls.
The Insomnia engineer has responded over there too. Will have to wait on them.

There's always https://github.com/Vyoam/InsomniaToPostmanFormat and (for non-sensitive cases) the online https://www.apimatic.io/transformer/#supported-formats

Still, I have made [an action item ](https://github.com/Vyoam/insomnia-plugin-postman-export/commit/a28935dc3fa03a21bb033cf46624e04cc7fd2891)to see what I should do better w.r.t showing plugin-related errors. Tho, for the final step (like for parsing errors etc.) I did observe them duly displayed during my development work.
